ui/app/styles/app.scss

```diff
@@ -2,147 +2,34 @@
-  --info-text: #{dehex(color.$blue-500)};
-}
-
-@media (prefers-color-scheme: dark) {
-  :root {
-    --brand: #{dehex($brand-color)};
-    --background: #{dehex(color.$black)};
-    --text: #{dehex(color.$ui-gray-100)};
-    --text-muted: #{dehex(color.$ui-gray-400)};
-    --text-subtle: #{dehex(color.$ui-gray-600)};
-    --link: #{dehex(color.$blue-400)};
-    --border: #{dehex(color.$ui-cool-gray-700)};
-    --outline: #{dehex(color.$ui-gray-700)};
-    --card: #{dehex(color.$ui-cool-gray-800)};
-    --panel: #{dehex(color.$ui-cool-gray-800)};
-    --shadow: #{dehex(color.$ui-cool-gray-900)};
-    --focus-ring: #{dehex(color.$blue-400)};
-    --badge: #{dehex(color.$ui-gray-800)};
-    --badge-text: #{dehex(color.$ui-gray-300)};
-    --success: #{dehex(color.$green-900)};
-    --success-text: #{dehex(color.$green-400)};
-    --success-border: #{dehex(color.$green-800)};
-    --error: #{dehex(color.$red-900)};
-    --error-text: #{dehex(color.$red-400)};
-    --error-border: #{dehex(color.$red-800)};
-    --warning: #{dehex(color.$yellow-900)};
-    --warning-text: #{dehex(color.$yellow-400)};
-    --warning-border: #{dehex(color.$yellow-800)};
-    --info: #{dehex(color.$blue-800)};
-    --info-text: #{dehex(color.$blue-200)};
-    --pds-color: #{dehex(color.$white)};
-    --pds-input-backgroundColor: #{dehex(color.$ui-cool-gray-900)};
-    --pds-input-color: #{dehex(color.$white)};
-  }
-}
-
-body {
-  @include Layout.stack(0);
-  color: rgb(var(--text));
-  background: rgb(var(--background));
-  min-height: 100vh;
-}
-
-// Additional resets
-
-p {
-  margin: 0;
-}
-
-ul {
-  list-style: none;
-  margin: 0;
-  padding: 0;
-}
-
-:any-link:not(.pds-button) {
-  border-bottom: none;
-}
-
-// Waypoint styles
-
-code {
-  font-family: ui-monospace, 'Menlo', monospace;
-  white-space: nowrap;
-  color: rgb(var(--text));
-  background: none;
-  border: none;
-  border-radius: 0;
-  padding: 0;
-}
-
-pre code {
-  white-space: pre;
-}
-
-hr {
-  margin: scale.$lg-1 0;
-  border: none;
-  border-top: 1px solid rgb(var(--border));
-}
-
-// Set a default size for icons
-
-svg.icon {
-  width: scale.$lg-2;
-  height: scale.$lg-2;
-  flex-shrink: 0;
-}
-
-.pds-icon {
-  display: inline-flex;
-  align-items: center;
-  justify-content: center;
-}
-
-@import './_footer';
-@import './_header';
-@import './_notifications';
-@import './_page';
-@import './pds-form';
-@import './status-badge';
-@import './x-toggle';
+@use 'pds/tokens/font';
+@use 'pds/core/typography/config' as Typography;
+@use 'utilities/animations';
+
+@import './variables';
+@import './layout';
+
+@import './components/app-card-item';
+@import './components/app-item';
+@import './auth-page';
+@import './components/badge';
+@import './components/navigation/breadcrumbs';
+@import './components/button';
+@import './components/card';
+@import './components/cli-hint';
+@import './components/copyable-code';
+@import './components/empty-state';
+@import './utilities/flash';
```

Comment:
this is much cleaner, but some of the folder structure could be improved. `utilities` should not contain components like the flash messages (even if the styles apply to an component that comes from an external addon). I'd suggest moving this under `/components`.